apps.json

```
@@ -3561,7 +3561,7 @@
     "unattended_upgrades": {
         "branch": "master",
         "category": "system_tools",
-        "level": 6,
+        "level": 0,
```

```suggestion
        "level": 6,
```

There is a regression in the ci, so I propose to keep the old level as there has been no change on this package for a while.